README.md

```diff
@@ -199,6 +199,7 @@ Julia does not install anything outside the directory it was cloned into. Julia
 * GCC version 4.7 or later is required to build Julia.
 * To use external shared libraries not in the system library search path, set `USE_SYSTEM_XXX=1` and `LDFLAGS=-Wl,-rpath,/path/to/dir/contains/libXXX.so` in `Make.user`.
   * Instead of setting `LDFLAGS`, putting the library directory into the environment variable `LD_LIBRARY_PATH` (at both compile and run time) also works.
+* It is important to note that the `USE_SYSTEM_*` flags should be used with caution. These are meant only for troubleshooting and porting, not for production use. Issues arising from the use of these flags will generally not be accepted.
```

Comment:
"not for production use" could imply that distribution packages using these flags would not work, which is contradictory with guidelines for packagers given elsewhere. Maybe "not for production use by most users"? Could also say "troubleshooting, porting and packaging".

Reply:
Nit: "It is important to note that" is redundant; "should be used with caution" ought to convey the necessary gravity of this remark. Besides, going straight to the point is more effective in terms of both taking in and retaining information.

Reply:
I believe our view is very clearly that we do not recommend using SYSTEM llvm and various other packages with known bugs. That seems to create a lot of friction when people discover those bugs and then report them, and are told that we do not support SYSTEM libraries.

Reply:
See recent discussion at https://github.com/JuliaLang/julia/pull/25912. Note that README now says:
> Package maintainers will typically want to make use of system libraries where possible. Please refer to the above version requirements and additional notes below. A list of maintained Julia packages for various platforms is available at https://julialang.org/downloads/platform.html.

That's why I suggest that we mention distribution packagers here for consistency.

(FWIW, I think the situation of distro packages is slowly improving, e.g. Arch has recently started to use our LLVM patches and IIRC they have also switched to OpenBLAS.)

Reply:
Ok. Good points. Let me rework that a bit.